app/i18n/it/conf.php

```diff
@@ -105,7 +105,13 @@
 	),
 	'profile' => array(
 		'_' => 'Gestione profili',
-		'api' => 'Gestione API',
+		'api' => array(
+			'_' => 'Gestione API',
+			'check_link' => 'Verificare lo stato dell'API tramite: <kbd><a href="../api" target="_blank">%s</a></kbd>',
```

Comment:
```suggestion
			'check_link' => 'Verificare lo stato dell\'API tramite: <kbd><a href="../api" target="_blank">%s</a></kbd>',
```
Sorry I didn't see it before

Reply:
```suggestion
			'check_link' => 'Verificare lo stato dell’API tramite: <kbd><a href="../api" target="_blank">%s</a></kbd>',
```

I saw that you want to use a different method from mine